src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCONT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCONT = 20;
+
+  let basePrice = days * PRICE_PER_DAY;
```

Comment:
Prefer 'const' over 'let' for the 'basePrice' variable since its value is not being reassigned later in the code.